Makefile

```diff
@@ -12,7 +12,7 @@ REGISTRY ?= ghcr.io
 USERNAME ?= talos-systems
 REGISTRY_AND_USERNAME ?= $(REGISTRY)/$(USERNAME)
 GOFUMPT_VERSION ?= abc0db2c416aca0f60ea33c23c76665f6e7ba0b6
-GO_VERSION ?= 1.14
+GO_VERSION ?= 1.16
```

Comment:
This and other build instructions files are managed by Kres, so it doesn't make sense to edit them manually, just `make rekres` :)